src/cascadia/TerminalSettingsEditor/ColorSchemesPageViewModel.idl

```diff
@@ -27,5 +27,7 @@ namespace Microsoft.Terminal.Settings.Editor
         Boolean CanDeleteCurrentScheme { get; };
         ColorSchemesSubPage CurrentPage;
         Windows.Foundation.Collections.IObservableVector<ColorSchemeViewModel> AllColorSchemes { get; };
+
+		void SchemeListItemClicked(IInspectable sender, Windows.UI.Xaml.Controls.ItemClickEventArgs args);

```

Comment:
code format?

Reply:
uh, code format doesn't fix idl files... and this looks like it _still_ landed with tabs in it instead of spaces.